### DISCUSSION TITLE
Could not find docker logs

### DISCUSSION THREAD

**Discussion by aberatakdogan** [cybertec-postgresql/pgwatch2#599] [Q&A]
Hi,

I am monitoring postgres db's in grafana by using a docker container which has a version 20.10.17.

Although in some databases I can observe replication lag data, in some of them I can not.

In this docker version, I can not see the reason of errors for "no data" dashboards in grafana. Where I can see the log of the daemon?

**Comment by pashagolub** [comment]
Hello, 

please find some hints on how to check Docker logs in out official documentation: https://pgwatch2.readthedocs.io/en/latest/docker_installation.html#interacting-with-the-docker-container
